### PR TITLE
[DTGB-490] Image wall implementatie

### PR DESCRIPTION
**Pull request by Jeroen005**
# Ticket
Link to ticket: https://digipolisgent.atlassian.net/browse/DTGB-490

### CODE REVIEW

File: source/js/image-wall.bindings.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  'use strict';
+
+  Drupal.behaviors.gentBaseLoadImageWall = {
+    attach: function (context, settings) {
+
+      if (!Macy) { // eslint-disable-line no-undef
+        return;
+      }
+
+      var list = document.getElementsByClassName('image-wall');
```
Comment (delrueba) [inline source/js/image-wall.bindings.js:15]: Why is this?
Does Macy break if the query returns no results?

File: styleguide/components/41-organisms/image-collections/image-wall/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Image gallery
```
Comment (delrueba) [inline styleguide/components/41-organisms/image-collections/image-wall/README.md]: Update title.

Thumbs up for including documentation!

**Review by meirege — APPROVED** [review]

**Review by delrueba — CHANGES_REQUESTED** [review]

File: source/js/image-wall.bindings.js (hunk 1 above)
Comment (Jeroen005) [reply source/js/image-wall.bindings.js:15]: Yups indeed, you get an console error when there are no image-wall elements on the page.

Comment (delrueba) [reply source/js/image-wall.bindings.js:15]: Damn, that's... bad code.
I'm sure someone will create an issue in the library :innocent:

**Review by delrueba — APPROVED** [review]

**Pull request merged**